uast/uast.go

```diff
@@ -220,6 +220,16 @@ const (
 	AssignmentVariable
 	AssignmentValue
 
+	// AugmentedAssignment is an augmented assignment usually combining the equal operator with
+	// another one (e. g. +=, -=, *=, etc). It is expected that children contains and
+	// AugmentedAssignmentOperator with a child or aditional role for the specific Bitwise or
+	// Arithmetic operator used. The AugmentedAssignmentVariable and AugmentedAssignmentValue roles
+	// have the same meaning than in Assignment
+	AugmentedAssignment
+	AugmentedAssignmentOperator
```

Comment:
Not sure about this, but... do we really need this constant? we already have the corresponding constants for operators right?